pkg/sanitization/sanitizer.go

```diff
@@ -19,10 +22,13 @@ func (s *Sanitizer) Apply(input string) string {
 	for _, rule := range s.rules {
 		output = rule.Pattern.ReplaceAllString(output, rule.Replacement)
 	}
+	if s.maxLength != 0 && s.maxLength < len(output) {
```

Comment:
Couldn't maxLength be achieved in regexp?
```go
Rule{
    Pattern: regexp.MustCompile(`(.{1,32}).*`),
    Replacement: "$1",
}
```

Reply:
it could, i think we need to actually put some effort into the sanitizer so i decided to just do this as a placholder